kernel/packages/decentraland-ecs/types/@decentraland/PortableExperiences/index.d.ts

```diff
@@ -0,0 +1,16 @@
+declare module '@decentraland/PortableExperiences' {
+  type PortableExperienceUrn = string
+  type PortableExperienceHandle = {
+    pid: PortableExperienceUrn
+    parentCid: string
+  }
+  type SpawnPortableExperienceParameters = {
+    urn: PortableExperienceUrn
+  }
+
+  export function spawn(spawnParams: SpawnPortableExperienceParameters): Promise<PortableExperienceHandle>
```

Comment:
Just curious, and to understand this better, how do you think `SpawnPortableExperienceParameters` will evolve? What else will it have, besides the urn?

Reply:
Changed to only one parameter